### DISCUSSION TITLE
Help! Disaster strikes

### DISCUSSION THREAD

**Discussion by Sl-a-ck** [OpenSageTV/Community#168] [Capture Devices]
Windows update killed SageTV server. So as an interim measure I have tried to install SageTV 9.2.6 server on other windows boxes. Win11. On 1 box it sees the hdhumerun flex 4k quad tuner, but channel scans within SageTV find nothing. Tuning a channel gets no signal. I can see the scan working the tuners via the tech app but the streaming bandwidth display 'network rate' remains at 0 bps all the time. The HDHomerun apps and the tech app find plenty of channels and they are watchable. The 'network rate' while watching via the hdhomerun app is 8-9 Mbps. I have tunred off the firewall. I recall an old dialog in these apps that had a selection for sage TV but canno find that in the lastest apps downloaded from silicon dust. On the second system also Win11. Server setup does not even see any capture devices available.  Really annoyed.

**Comment by Ugadata** [comment]
Some versions of the HDHomeRun Setup only recognize the 32-bit version of Sage.  This has been fixed so both 32 and 64 will be recognized in the more recent versions of HDHR Setup (current version 20231214).  Your looking for the BDA Compatibility Mode (Configuration Tab of the HDHR Setup application).  If you can't set it to SageTV , You should be able to use the JRiver (I think).

Did you do a scan in the HDHR Setup software before doing a scan in Sage?

**Comment by TGsell1** [comment]
I’m not sure this will help you this time but I always make a disc image prior to a Windows update or I feeeze updates indefinitely for a server that doesn’t do anything but Sage.  Sent from my iPhoneOn Nov 3, 2024, at 11:18 AM, Sl-a-ck ***@***.***> wrote:﻿
Windows update killed SageTV server. So as an interim measure I have tried to install SageTV 9.2.6 server on other windows boxes. Win11. On 1 box it sees the hdhumerun flex 4k quad tuner, but channel scans within SageTV find nothing. Tuning a channel gets no signal. I can see the scan working the tuners via the tech app but the streaming bandwidth display 'network rate' remains at 0 bps all the time. The HDHomerun apps and the tech app find plenty of channels and they are watchable. The 'network rate' while watching via the hdhomerun app is 8-9 Mbps. I have tunred off the firewall. I recall an old dialog in these apps that had a selection for sage TV but canno find that in the lastest apps downloaded from silicon dust. On the second system also Win11. Server setup does not even see any capture devices available.  Really annoyed.

**Comment by Sl-a-ck** [comment]
I managed to get a server up and running. The HD Homerun windows install (not the app store app, and not the 'tech' app) is required to set the dba profile (or terms to that effect) to 'SageTV' after that, I did a scan in HD Homerun, and subsequent scan in Sage setup found channels. Just in time for NFL recordings.

And yeah I should have had an image. But something else is afoot with the old box. After trying all manner  of options with rescue media. I proceeded with a clean install. After the bootable usb stick does it's thing and reboots - it never boots from the hard disk (Samsung SDD). It just boots again from the USB stick.

**Comment by KryptoNyte-Sage** [comment]
I usually pull the USB stick after it's evident that the Windows install is done with the install media.

**Reply by Sl-a-ck** [reply]
Yeah did that.

**Comment by TGsell1** [comment]
Aaah yes… image won’t help that.   You’ll figure it out.  Sent from my iPhoneOn Nov 3, 2024, at 6:10 PM, Sl-a-ck ***@***.***> wrote:﻿
I managed to get a server up and running. The HD Homerun windows install (not the app store app, and not the 'tech' app) is required to set the dba profile (or terms to that effect) to 'SageTV' after that, I did a scan in HD Homerun, and subsequent scan in Sage setup found channels. Just in time for NFL recordings.
And yeah I should have had an image. But something else is afoot with the old box. After trying all manner  of options with rescue media. I proceeded with a clean install. After the bootable usb stick does it's thing and reboots - it never boots from the hard disk (Samsung SDD). It just boots again from the USB stick.

**Comment by Sl-a-ck** [comment]
I'll pull the os disk and run Samsung's SSD diagnostic on it. Recordings are on rotating media, and they are backed up. So I should be able to get them back, but it's just tv. First world problems. The big scare was not getting something functional by kick off and having to watch that live.

**Comment by Sl-a-ck** [comment]
2 Things: Boot disk is dying, and sage recordings disk is dead, well not getting a drive letter. I've validated it's not a SATA and power connectors issue as they work on another drive.

Pulled the boot disk. Seems fine on a USB/Sata adapter. Copied the SageTV folder from windows.old.000 There were (3) of them. .old .old.00 and .old.001. Samsung magician says it's fine. Did a quick scan (s.m.a.r.t not available, I guess 'cuz USB) on the disk and no issues reported. 

Put it back in the old SageTV box, and it boots doing a windows startup/install. Finish the install.

I download macrium reflect thinking I have one license available I'll do a nightly image. It's aborting backups because reading fails...CRC.
So I install Samsung magician on the new install it's flagging LBA failures. Guess I'm getting a new boot disk in addition to the surveillance disk I ordered to be the primary recordings holder. Good thing I have a copy of the recordings folder as I had a sync app syncing BigSage1 -> BigSage2 nightly.
